### PR TITLE
Added a decorated fir tree and a present

### PR DESCRIPTION
**Pull request by shaeone**
## About the PR 
I created two sprites for the next holidays as requested on the Discord. The first one is a decorated tree with animated lights and the second one is a present box.

The tree could be added in the maps for the next days manually, or let players build it up with some recipes in-game.

The idea for the present is that somehow one present is delivered to each player online (automatically or delivered by cargo), so the player can open it to get a ramdom "Fun" category object (bike horns, soaps, crayons, toys...). The present could change to the "unwrapped" state, looking like a piece of paper without use once opened.

Unfortunately, I don't have enough knowledge to code this.

**Screenshots**

https://gyazo.com/6dbf036b902b4c666f00b85a82312b7f

**Changelog**

:cl:
- add: Added a decorated tree and presents!

### CODE REVIEW

**Comment by moonheart08** [comment]
I can code the present itself when this is merged.

File: Resources/Textures/Objects/Decoration/present.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made with love by Shaeone",
```
Comment (moonheart08) [inline Resources/Textures/Objects/Decoration/present.rsi/meta.json:4]: Need an actual license. Maybe CC-BY-NC-SA-3.0

File: Resources/Prototypes/Entities/Structures/Decoration/decorated_fir_tree.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+- type: entity
+  id: Decorated Fir Tree
+  parent: BaseStructure
+  name: Decorated Fir Tree
+  description: It makes happy most of the people.
```
Comment (ElectroJr) [inline Resources/Prototypes/Entities/Structures/Decoration/decorated_fir_tree.yml:5]: The grammar here is a bit weird

**Comment by ElectroJr** [comment]
The tree looks great, but in stark contrast to the tree the un-wrapped present just looks like a weird uniform-colour blob, and IMO it is really not clear what it is meant to be (just a crumpled pile of wrapping paper?).

Also, for the change log: given that the tree & presents haven't yet been mapped in or are somehow obtainable w/o admin intervention I'm worried players will read the change-log and start trying to find/get the tree and then be disappointed that it's not yet "actually added". I'm really not sure about how that's usually handled given you should be properly credited in the in-game changelog, so input from other maintainers is probably needed.

**Review by moonheart08 — APPROVED** [review]
lgtm

File: Resources/Prototypes/Entities/Structures/Decoration/decorated_fir_tree.yml (hunk 2 above)
Comment (moonheart08) [reply Resources/Prototypes/Entities/Structures/Decoration/decorated_fir_tree.yml:5]:
```suggestion
  description: A very festive tree for a very festive holiday.
```

**Comment by shaeone** [comment]
I've changed the present PNGs to give it more contrast:
image

**Review by ElectroJr — APPROVED** [review]

**Pull request merged**